assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java

```diff
@@ -1679,6 +1679,7 @@ public class SimpleSubmission {
         private String gradableId;
         private String submittedText;
         private String dateSubmitted;
+        private Long dateSubmittedTimestamp;
```

Comment:
One final thing, I promise. I think it would make more sense to name this field as dateSubmittedEpochSeconds. Then, it is perfectly clear what it is for anybody using the JSON api.

Reply:
No problem! Much clear :D